### DISCUSSION TITLE
UCB CS61B: Data Structures and Algorithms - CSDIY.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#325] [Announcements]
# UCB CS61B: Data Structures and Algorithms - CSDIY.wiki

CS自学指南

https://csdiy.wiki/en/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E4%B8%8E%E7%AE%97%E6%B3%95/CS61B/

**Comment by VidhyaVarshanyJS** [comment]
I wis to learn this course to learn DSA ..how can I join ? I don't get the invite code..

**Comment by Kolucci69** [comment]
If you want to learn you click on the link upstairs and you will see what happens. Good luck love,KOLUCCI .

**Comment by VidhyaVarshanyJS** [comment]
Which link should I need to click?

**Reply by sigsegv0x3f** [reply]
Click the course website link. There is an announcement which tells you the code :)

**Comment by VidhyaVarshanyJS** [comment]
MNXYKX this is not working ...

**Reply by VidhyaVarshanyJS** [reply]
Is this 2022 course?

**Reply by sigsegv0x3f** [reply]
Try P5WVGW from the 2021 website: https://sp21.datastructur.es/about.html

**Reply by VidhyaVarshanyJS** [reply]
I am getting invalid code in the gradescope...

**Reply by sigsegv0x3f** [reply]
Try MB7ZPY for the both 2021 and 2018 gradescore websites. Source: https://www.reddit.com/r/berkeley/comments/ro1qdy/cs61b_entry_code/

**Reply by VidhyaVarshanyJS** [reply]
I can only see assignments and projects..I could not access lecture videos...

**Reply by sigsegv0x3f** [reply]
Gradescope is usually used for autograding of assignments and projects. You don't watch lecture videos on gradescope Imao. The lecture videos are available on YouTube. Try visiting Josh Hug's YouTube channel

**Reply by VidhyaVarshanyJS** [reply]
Thanks for your time. Can you link the Google docs on guide to use this cs61 ..GitHub guide?

**Reply by sigsegv0x3f** [reply]
I don't understand your question. You mean how to use this cs diy wiki page? Or you want to know how to see the course matrials of cs61b?

**Comment by VidhyaVarshanyJS** [comment]
Both of them..

**Reply by sigsegv0x3f** [reply]
As a computer science student, you need to learn to search and read the docs by yourself. I believe the course website has all the video links, lecture slides you want. And this cs wiki page is just a summary of resource links and are quite self explanatory...

**Reply by VidhyaVarshanyJS** [reply]
Got it !
.To say this I'm novice..

**Reply by VidhyaVarshanyJS** [reply]
Thanks for the time and help 😌

**Comment by HsHs-dev** [comment]
2021 iteration (currently in progress..)
https://github.com/HsHs-dev/UCB-CS61B
appreciate any advices/issues/improvements/collaboration
